myst_nbs/gaussian_processes/GP-Latent.myst.md

```diff
@@ -6,50 +6,43 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python (PyMC3 Dev)
+  display_name: pymc
   language: python
-  name: pymc3-dev
+  name: pymc
 ---
 
+```{raw-cell}
```

Comment:
This should not be a raw cell, it should be markdown cell. It is not rendering correctly in the preview due to that: https://pymcio--371.org.readthedocs.build/projects/examples/en/371/gaussian_processes/GP-Latent.html. The title is not shown, neither are the notebook badges and the tags are not recognized.